samples/peripheral/radio_test/README.rst

```diff
@@ -130,6 +132,29 @@ User interface
    * - transmit_pattern
      - <sub_cmd>
      - Set transmission pattern.
+   * - total_output_power
+     - <tx output power>
+     - Set total output power in dBm.
+       This value includes SoC output power and front-end module gain.
+
+Tx output power
+===============
+
+This sample has a few different command that you can use to test device output power.
+The commands behavior varies depending on hardware configuration and Kconfig options.
+
+1. Radio Test without front-end module support:
+   The ``output_power`` command sets SoC output command with subcommands set.
+   The output power is set directly in the RADIO peripheral.
```

Comment:
```suggestion
* Radio Test without front-end module support:
  * The ``output_power`` command sets the SoC output command with a subcommand set.
    The output power is set directly in the radio peripheral.

```
Changed to bullet list, because this is not a sequence of steps.